### PR TITLE
Fixed Smoke test failure

### PR DESCRIPTION
**Pull request by pravin-deriv**
In this PR
- I have moved euHomeHamburger.cy.js to smoke folder
- I have fixed few of the test scripts that failed due to recent revamp

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = defineConfig({
   e2e: {
     projectId: '16kef2',
     setupNodeEvents(on, config) { },
-    //baseUrl: 'https://deriv.com',
+    // baseUrl: 'https://deriv.com',
```
Comment (markodonnell-deriv) [inline cypress.config.js:11]: Best to remove comments. I think staging-app.deriv.com is a reasonable url to commit.

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Looks good with 1 minor comment.
Good use of force: true, by the way. It often comes in handy!

File: cypress.config.js (hunk 1 above)
Comment (habib-deriv) [inline cypress.config.js:11]:
```suggestion
```

**Review by habib-deriv — APPROVED** [review]
LGTM

File: cypress.config.js (hunk 1 above)
Comment (habib-deriv) [inline cypress.config.js]:
```suggestion
```

**Review by markodonnell-deriv — APPROVED** [review]
Looks good.

File: cypress.config.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ module.exports = defineConfig({
   e2e: {
     projectId: '16kef2',
     setupNodeEvents(on, config) { },
-    //baseUrl: 'https://deriv.com',
     //baseUrl: 'https://deriv-com-v2.pages.dev/',
```
Comment (markodonnell-deriv) [inline cypress.config.js:11]: Remove comment

**Review by markodonnell-deriv — APPROVED** [review]
Looks good.

**Pull request merged**